core/build.gradle.kts

```diff
@@ -26,4 +26,10 @@ dependencies {
     api(libs.bundles.jackson)
 
     testImplementation(libs.junit4)
```

Comment:
Is it easy to mix tests written against both the 4 and 5 APIs within the same module as we migrate? We should set up a policy, automated if possible, that new tests be written against the Jupiter API.

Reply:
yup, the Redshift test I migrated over to 5 is an example of that working (everything else in that package is junit4). it should be relatively straightforward to use detekt enforce usage of 5 in new test _classes_, but we should spend some time writing compatibility shims for our junit rules before enforcing that policy